earthfile2llb/listener.go

```diff
@@ -280,6 +281,9 @@ func (l *listener) ExitCopyStmt(c *parser.CopyStmtContext) {
 		return
 	}
 	if allArtifacts {
+		if dest == "" || dest == "." || len(srcs) > 1 {
+			dest += string(filepath.Separator)
```

Comment:
Thankyou for using filepath. One less place to touch when we go to Windows.

Reply:
I'm not really convinced that it's the best use though :-) . Because `filepath` is host platform dependent. Whereas inside the Earthfile, we perhaps want to be guest-platform dependent. So there's still stuff to tweak.